k8s/commander.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -exo pipefail
+set -eo pipefail
```

Comment:
Let's keep the `x` here

Reply:
the `x` here is just polluting the output and not giving any useful information. Bash -x is used for debugging scripts.

Anyway, I'm taking the suggestion because don't want to fight this battle :)